src/main/scala/SparkER/Utilities/StatisticsEstimator.scala

```diff
@@ -300,7 +298,7 @@ object StatisticsEstimator {
   /** *
     * Estimates the blocks entropy
     * */
-  def estimateNormalizedEntropy(blocks: RDD[BlockAbstract], profileBlocks: Broadcast[Map[Long, ProfileBlocks]]): RDD[(Long, Double)] = {
+  def estimateNormalizedEntropy(blocks: RDD[BlockAbstract], profileBlocks: Broadcast[Map[Long, ProfileBlocks]]): RDD[(Int, Double)] = {

```

Comment:
Changed Long to Int, is this intentional ?

Reply:
yes, there will be compilation errors in scala 2.12 in case it is `Long`. `blockID` is `Int` which is returning as part of RDD, hence this change was done.